commit fd094ba7f4712c577cc6c7123532eb3e7044af98
Author: Alexander Mezin <mezin.alexander@gmail.com>
Date:   Sun Dec 22 22:54:14 2013 +0700

    Fix memory leak in XRandrBrightness::brightness
    
    "break" jumps to the end of the method, and "info"
    is never freed.
    Add missing XFree(info).

diff --git a/powerdevil/daemon/backends/upower/xrandrbrightness.cpp b/powerdevil/daemon/backends/upower/xrandrbrightness.cpp
index 3c77001..f18092e 100644
--- a/powerdevil/daemon/backends/upower/xrandrbrightness.cpp
+++ b/powerdevil/daemon/backends/upower/xrandrbrightness.cpp
@@ -107,6 +107,7 @@ float XRandrBrightness::brightness() const
                 {
                     double min = info->values[0];
                     double max = info->values[1];
+                    XFree(info);
 
                     // FIXME for now just return the first output's value
                     result = (cur - min) * 100 / (max - min);
